<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackerOutputViewer</class>
 <widget class="QWidget" name="TrackerOutputViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracker output viewer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>23</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Filename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tracker_output_filename">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Dataset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tracker_output_dataset">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Walltime</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tracker_output_walltime">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Session selection</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QPushButton" name="tracker_output_prev_session">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressource.qrc">
              <normaloff>:/Button-Previous-icon.png</normaloff>:/Button-Previous-icon.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Session id</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="tracker_output_next_session">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="ressource.qrc">
              <normaloff>:/Button-Next-icon.png</normaloff>:/Button-Next-icon.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="tracker_output_session_id_combobox"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="tracker_output_sync_to_session">
            <property name="text">
             <string>Sync to dialog session-id</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Status </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tracker_output_status_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="tracker_output_tree_view">
       <column>
        <property name="text">
         <string>Turns</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
